en/mkdocs.yml

```diff
@@ -89,15 +83,16 @@ markdown_extensions:
   - pymdownx.keys
   - pymdownx.mark
   - pymdownx.smartsymbols
-  - pymdownx.superfences
```

Comment:
Why are we removing this extension?

Reply:
This conflicts with highlightjs, it intends to use pygments instead